services/gitea/gitea-forks.tester.js

```diff
@@ -0,0 +1,33 @@
+import Joi from 'joi'
+import { isMetric } from '../test-validators.js'
+import { createServiceTester } from '../tester.js'
+
+export const t = await createServiceTester()
+
+t.create('Forks')
+  .get('/gitea/tea.json')
+  .expectBadge({
+    label: 'forks',
+    message: isMetric,
+    color: 'blue',
+    link: ['https://gitea.com/gitea/tea', 'https://gitea.com/gitea/tea/forks'],
+  })
+
+t.create('Forks (self-managed)')
+  .get('/CanisHelix/shields-badge-test.json?gitea_url=https://codeberg.org')
+  .expectBadge({
+    label: 'forks',
+    message: Joi.number().integer(),
```

Comment:
why not `isMetric` here?

Reply:
**isMetric** only accepts >0 and I do not have any forks of the CI test project. Trying to avoid creating dummy accounts on https://codeberg.org where possible, but if this is preferred I could do it. The risk is that the 2nd dummy account could get purged due to external factors.

Saying that. **isMetricAllowNegative** seems to work.

Reply:
Can we use a repo that has >0 forks for the test
e.g: https://codeberg.org/Codeberg/Documentation